api/types/src/transaction.rs

```diff
@@ -2399,3 +2399,23 @@ pub struct GasEstimation {
     /// The prioritized estimate for the gas unit price
     pub prioritized_gas_estimate: Option<u64>,
 }
+
+/// API representation of persisted auxiliary transaction information
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
+pub struct PersistedAuxiliaryInfo {
```

Comment:
Should we make it an enum here?

Reply:
This struct is used for JSON output. I think we can add more fields here without breaking. 
I've added replay_protection_nonce field to UserTransactionRequest struct earlier without breaking anything.

Reply:
I saw in your code the accept_type can be BCS, no?